src/add.js

```diff
@@ -20,40 +17,74 @@ form.addEventListener('submit', (e) => {
   e.preventDefault();
 });
 
-function addScore() {
-  const addButton = document.querySelector('.add');
-  addButton.addEventListener('click', () => {
-    const name = inputName.value;
-    const score = inputScore.value;
-    if (name === '' || score === '') {
-      return;
-    }
-    const data = new Score(name, score);
-    recentScore.push(data);
-    localStorage.setItem('scores', JSON.stringify(recentScore));
-    palmares(data);
-    scoreContainer.style.border = '2px solid #000';
-    clearField();
-    refresher();
-  });
-}
-addScore();
-
 function palmares(data) {
-  scoreContainer.innerHTML += `<li class="stat"><span class="player-name">${data.name}:</span>
+  scoreContainer.innerHTML += `<li class="stat"><span class="player-name">${data.user}:</span>
                                    <span class="player-score">${data.score}</span></li>`;
 }
 
-recentScore.forEach((score) => {
-  palmares(score);
-  scoreContainer.style.border = '2px solid #000';
-});
-
 function clearField() {
   const name = inputName;
   const score = inputScore;
   name.value = '';
   score.value = '';
 }
 
-refresher();
\ No newline at end of file
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const gameId = () => {
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: 'christian game' }),
+  }).then((response) => response.json()).then((data) => {
+    localStorage.setItem('gameId', JSON.stringify(data));
+  });
+};
+gameId(url);
+
+const urlExample = `${url}TmfTBSDlBvHPPY995ITG/scores`;
+
+async function addGame(urlExample, newGame) {
+  await fetch(urlExample, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(newGame),
+  }).then((response) => response.json());
+}
+
+async function getData(url) {
+  const a = await fetch(url);
+  const b = await a.json();
+  recentScore = b.result;
+  recentScore.forEach((score) => {
+    palmares(score);
+  });
+  localStorage.setItem('scores', JSON.stringify(recentScore));
```

Comment:
- [ ] please make use of ES6 arrow functions instead of the function keyword as per the project requirements.

Reply:
Thanks, @mikethreels! I almost forget every time to use the arrow function :smiley: